MathLib/MathematicalConstant.h

```diff
@@ -0,0 +1,20 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2020, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ * Created on June 4, 2020, 5:09 PM
+ */
+
+#pragma once
+
+namespace MathLib
+{
+namespace MathematicalConstant
+{
+constexpr double pi = 3.14159265359;
```

Comment:
Use boost for constants. No need for 25th version of that number.

Reply:
Thanks.